source/includes/connection-snippets/x509-snippets/nodejs-connection.js

```diff
@@ -0,0 +1,22 @@
+// begin x509 connection
+const { MongoClient } = require('mongodb');
+const fs = require('fs');
+
+const client = new MongoClient('mongodb+srv://<cluster-url>?authMechanism=MONGODB-X509&ssl=true', {
+  sslCA: fs.readFileSync('/etc/certs/mongodb/ca.pem'),
+  sslKey: fs.readFileSync('/etc/certs/mongodb/client.pem'),
+  sslCert: fs.readFileSync('/etc/certs/mongodb/client.pem')
```

Comment:
@mbroadst should this use tls URI options instead (per [DRIVERS-557](https://jira.mongodb.org/browse/DRIVERS-557))?

Reply:
The latest version of the node-docs use `ssl` (see: https://mongodb.github.io/node-mongodb-native/3.3/reference/connecting/connection-settings/)